### PR TITLE
Feature/standard

### PR DESCRIPTION
**Pull request by millerts**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -50,7 +50,7 @@ dependencies {
     compileOnly 'com.google.code.findbugs:annotations:3.0.1'
     compile 'org.hibernate:hibernate-java8:5.0.12.Final'
     compile 'org.opentestsystem.ap:ap-common:0.3.32'
-    compile 'org.opentestsystem.ap:ap-imrt-common:0.1.17'
+    compile 'org.opentestsystem.ap:ap-imrt-common:0.1.20-SNAPSHOT'
```
Comment (millerts) [inline build.gradle]: Will update once common is released

File: src/main/java/org/opentestsystem/ap/imrt/iis/model/ItemRevisionImpl.java [hunk 2]
```diff
@@ -51,7 +46,7 @@ private void createDomainModelObjects() {
         findMetadata();
         createImrtItem();
         createItemGitInformation();
-        createStandardIds();
+        updateItemStandardIds();
```
Comment (millerts) [inline src/main/java/org/opentestsystem/ap/imrt/iis/model/ItemRevisionImpl.java]: I renamed this method as it now updates the item.

File: src/main/java/org/opentestsystem/ap/imrt/iis/model/ItemRevisionImpl.java [hunk 3]
```diff
@@ -130,111 +125,54 @@ private void createItemGitInformation() {
...
-                    .withClaim(metadata.getPrimaryClaim())
-                    .withContentDomain(metadata.getContentDomain())
-                    .withTarget(metadata.getPrimaryAssessmentTarget())
-                    .withEmphasis("")  //TODO - IAT required to populate this information
-                    .withCommonCoreStandard(metadata.getPrimaryCommonCoreStandard())
-                    .withUpdatedBy(itemBankItemRevision.getAuthor())
-                    .build());
-        }
+    private void updateEnglishItemStandardId(BaseItem item) {
+        item.setPrimaryClaim(StringUtils.defaultString(metadata.getPrimaryClaim()));
```
Comment (millerts) [inline src/main/java/org/opentestsystem/ap/imrt/iis/model/ItemRevisionImpl.java]: ELA items only have a primary claim.  I went back and forth if we should always just populate it it all now that these are stored differently.

File: src/main/java/org/opentestsystem/ap/imrt/iis/model/ItemRevisionImpl.java [hunk 4]
```diff
@@ -130,111 +125,54 @@ private void createItemGitInformation() {
...
+        item.setPrimaryTarget(StringUtils.defaultString(metadata.getPrimaryAssessmentTarget()));
+        item.setPrimaryContentDomain(StringUtils.defaultString(metadata.getContentDomain()));
```
Comment (millerts) [inline src/main/java/org/opentestsystem/ap/imrt/iis/model/ItemRevisionImpl.java]: @jeffjohnson9046 - IAT stores content domain two different ways even though they are the same thing.

File: src/main/java/org/opentestsystem/ap/imrt/iis/model/ItemRevisionImpl.java (hunk 3 above)
Comment (jeffjohnson9046) [reply src/main/java/org/opentestsystem/ap/imrt/iis/model/ItemRevisionImpl.java]: I kinda like the idea of always setting the primary claim, then setting the other claims for a MATH item.  Maybe something like:

```java
private void updateStandardIds() {
    if (metadata == null) {
        return;
    }

    final String contentDomain = metadata.getSubject().equalsIgnoreCase(ItemConstants.ItemSubject.SUBJECT_ELA)
         ? StringUtils.defaultString(metadata.getContentDomain())
         : StringUtils.defaultString(metadata.getPrimaryContentDomain());

    // set primary claim here regardless of subject, and maybe some sort of comment indicating that ELA only ever has a primary claim.
    item.setPrimaryClaim(StringUtils.defaultString(metadata.getPrimaryClaim()));
    item.setPrimaryTarget(StringUtils.defaultString(metadata.getPrimaryAssessmentTarget()));
    item.setPrimaryContentDomain(contentDomain);
  item.setPrimaryCommonCoreStandard(StringUtils.defaultString(metadata.getPrimaryCommonCoreStandard());

    if (ItemConstants.ItemSubject.SUBJECT_MATH.equalsIgnoreCase(metadata.getSubject())) {
        updateMathItemStandardId(baseItem); // <-- could probably rename this to "updateSubsequentStandardId"
    }
}
```

File: src/main/java/org/opentestsystem/ap/imrt/iis/model/ItemRevisionImpl.java (hunk 4 above)
Comment (jeffjohnson9046) [reply src/main/java/org/opentestsystem/ap/imrt/iis/model/ItemRevisionImpl.java]: I'm not sure I understand this comment, but I'm not intimately familiar with the data yet.  I think that's something we can discuss offline.

Comment (jeffjohnson9046) [reply src/main/java/org/opentestsystem/ap/imrt/iis/model/ItemRevisionImpl.java]: ...and now I do.  Ignore my previous comment.

File: src/main/java/org/opentestsystem/ap/imrt/iis/model/ItemRevisionImpl.java [hunk 5]
```diff
@@ -130,111 +125,54 @@ private void createItemGitInformation() {
      * Creates the standardId objects if the associated data is present.
```
Comment (jeffjohnson9046) [inline src/main/java/org/opentestsystem/ap/imrt/iis/model/ItemRevisionImpl.java]: I think this should read `Updates the standardId fields....`

File: src/main/java/org/opentestsystem/ap/imrt/iis/model/ItemRevisionImpl.java (hunk 3 above)
Comment (millerts) [reply src/main/java/org/opentestsystem/ap/imrt/iis/model/ItemRevisionImpl.java]: Got it.  I'll make this change.

**Pull request merged**